packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -25,6 +25,12 @@
   border: 1px solid transparent;
   background-color: $color-primary-base;
   box-shadow: 0 0 0 0 $color-primary-light;
+  color: $white;
+  font-family: $body-font-family;
+  font-size: ms(0);
+  font-weight: 600;
+  line-height: $base-line-height;
+  white-space: nowrap;
```

Comment:
If we use `<Box as="button">` rather than styling an instrinsic `<button />` element can we remove some amount of the duplicated font, line-height, whites-space styles yes? I thought that was the direction we were moving so that children args and others inherit font styles naturally.

We *can* redefine some of this for other components but I'd be curious to know why and what the benefit is?

Also consider that declarations like `background-color` below can also be removed assuming they have a corresponding variants defined in `_box-background.scss`. We'd just need to pass the correct prop to Box at our root of Button. Allowing us to build a more natural API for button ontop of Box, e.g. `variant="danger"` for Button as opposed to both `backgroundColor="danger-light"` and `color: "white"` passed to Box.

Reply:
+1 to using Box as Button's root to help remove some duplicate styles.

Re: Box for Button's `background-color`, we'd need to build out `:hover` and `:focus` state utility classes for Box. Might get yucky pretty quickly.

Reply:
Sure, and yes I'd much prefer sticking with class based interactivity until we know more about utility usage.

Reply:
Ok, the issue that we encounter here is that Button and Box root styles conflict. Button wants a white text that is semi bolded. Since both classes have a strength of 1 import order has Box trumping Button. If we append props for those text values we again enter into interactive class territory.

I'd prefer to address this issue in its own card for Button using Box

Reply:
I'm good with that. I didn't realize we hadn't solved for Box + `className` property overlap - definitely out of scope for this PR.